AnyText/AnyText.Core/Rules/LiteralRuleApplication.cs

```diff
@@ -77,6 +77,16 @@ protected virtual void OnMigrate(string oldValue, string newValue, ParseContext
         {
             if (oldValue != newValue)
             {
+                if (context.GetDefinition(oldValue).CurrentPosition.CompareTo(CurrentPosition) == 0)
```

Comment:
Wenn wir statt dem Identifier direkt das semantische Element verwenden, können wir hierdrauf glaube ich verzichten. Das semantische Element ändert sich ja nicht.

Reply:
Plus, es sieht ziemlich schräg aus, ich würde diese Logik keinesfalls in der `LiteralRuleApplication` erwarten